### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by uday-nv-work**
Added lists component

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Clean code
good to merge

File: component/Lists/lists.css
Comment (bhushannaikaroskar) [inline component/Lists/lists.css:42]: you can use px values here as value seem too small

File: component/Lists/lists.css
Comment (bhushannaikaroskar) [inline component/Lists/lists.css:22]: please make proper spacing between two class declarations for easy readability

**Review by bhushannaikaroskar — APPROVED** [review]
Hey Uday, good can clean code. Liked the way you have separated list styles for different types. I have given some  suggestions please check before merging. Approving the merge provided that you make these small changes.

**Review by zoyatanweer — APPROVED** [review]
- clean code, easy to understand
- do the suggested changes and you're good to merge
- good work

**Pull request merged**
